src/main/kotlin/no/nav/personbruker/dittnav/eventaggregator/informasjon/InformasjonRepository.kt

```diff
@@ -0,0 +1,22 @@
+package no.nav.personbruker.dittnav.eventaggregator.informasjon
+
+import kotlinx.coroutines.runBlocking
+import no.nav.personbruker.dittnav.eventaggregator.database.Database
+import no.nav.personbruker.dittnav.eventaggregator.database.entity.Informasjon
+import no.nav.personbruker.dittnav.eventaggregator.database.entity.createInformasjon
+
+class InformasjonRepository(private val database: Database) {
```

Comment:
Denne klassen er muligens litt midlertidig, men den ble opprettet slik at database-logikk kunne fjernes fra InformasjonEventService. Dermed kunne InformasjonEventService testes på en god måte, uten å forholde seg til masse database-spesifikke ting.

NB! InformasjonRepository vanskelig å teste, fordi det er så mange lambda-uttrykk som er nøstet i hver andre. Men alle delene som er nøstet sammen her er enhetstestet hver for seg, der de er implementert.